src/components/common/styles/Sidebar.css

```diff
@@ -4,10 +4,6 @@
   justify-content: space-between;
 }
 
-.ant-switch-checked {
-  background-color: #BFBFBF !important;
```

Comment:
yay for removing '!important' !! :) I love that you're not using ultimatum coding here.

Reply:
Just realized that this is the code that was interfering with your toggle. Good find!